ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/l3-port-channel-sub-interface-with-member-interfaces.yml

```
@@ -0,0 +1,45 @@
+---
+type: wan_router
+wan_mode: cv-pathfinder
+wan_router:
+  nodes:
+    - bgp_as: 65000
+      name: l3-port-channel-sub-interface-with-member-interfaces
+      id: 1
+      loopback_ipv4_pool: 192.168.0.0/24
+      vtep_loopback_ipv4_pool: 192.168.1.0/24
+      l3_interfaces:
+        - name: Ethernet5
+          wan_carrier: FOO
+          ip_address: 172.16.0.3/30
+      l3_port_channels:
+        - name: Port-Channel5
+          peer: peer2
+          peer_port_channel: Port-Channel15
+          peer_ip: 192.168.1.15
+          member_interfaces:
+            - name: Ethernet1/19
+              peer_interface: Ethernet1/19
+            - name: Ethernet1/20
+              peer_interface: Ethernet1/20
```

I think you don't need an extra l3_interfaces - the reason why you have it os probably because you needed a WAN interface for the wan_router config to be valid. We probably can even replace this to be an l3_leaf given that the test is only about the member

```suggestion
      l3_port_channels:
        - name: Port-Channel5
          peer: peer2
          wan_carrier: FOO
          peer_port_channel: Port-Channel15
          peer_ip: 192.168.1.15
          member_interfaces:
            - name: Ethernet1/19
              peer_interface: Ethernet1/19
            - name: Ethernet1/20
              peer_interface: Ethernet1/20
```